src/biblenotify/core/notifications.py

```diff
@@ -96,7 +98,43 @@ def loadChapter(self, location: str) -> dict:
 
         contents_json = json.loads(contents_string)
 
+        text = self.highlightVerses(contents_json)
+
         return {
-            "text": contents_json["read"][0]["text"],
+            "text": text,
             "place": contents_json["read"][0]["chapter"]
         }
+
+    @Slot(str, result="QVariant")
+    def highlightVerses(self, contents_json):
+        # assuming every second verse is preceded by a special delimeter ;
+        # another delimeter should be used since this delimeter have too many edge cases
+        list = re.split(r";", verse["verse"])
+        start = list[0]
+        end =  list[-1]
+
+        # to find index of displayed verse from global dictionary {verse} in {contents_json}
+        slice1 = re.search(start, contents_json["read"][0]["text"]).span()
+        slice2 = re.search(end, contents_json["read"][0]["text"]).span()
+
+        to_bold = contents_json["read"][0]["text"][slice1[0] : slice2[1]]
+
+        # in case of two verses
+        if re.search("</p><p><sup>\d</sup>",to_bold):
+            number = re.findall(r"\d",to_bold)
+
+            split_pattern = r'</p><p><sup>\d</sup>'
+            separator = '</b></p><p><sup>{}</sup><b>'
+            split_result = re.split(split_pattern, to_bold)
+
+            # joining inorder to get <b> tag inside of <p> tag and substituting number of second verse
+            formatted_string = separator.join(split_result).format(*number)
+
+        # in case of one verses
+        else:
+            formatted_string = to_bold
+
+
+        text = contents_json["read"][0]["text"].replace(to_bold, "<b>" + formatted_string + "</b>")
```

Comment:
Since we're passing in ``chapter`` all instances of ``contentsJson["read"][0]["text"]`` should be ``chapter``.

Reply:
Alright